Source/WebCore/Modules/mediastream/PeerConnectionBackend.cpp

```diff
@@ -446,7 +446,8 @@ void PeerConnectionBackend::generateCertificate(Document& document, const Certif
         return;
     }
 
-    LibWebRTCCertificateGenerator::generateCertificate(document.securityOrigin(), page->libWebRTCProvider(), info, [promise = WTFMove(promise)](auto&& result) mutable {
+    auto& webRTCProvider = reinterpret_cast<LibWebRTCProvider&>(page->webRTCProvider());
```

Comment:
A static_cast should work to, given LibWebRTCProvider probably derives from WebRTCProvider.